### PR TITLE
Feature/modules

### PR DESCRIPTION
**Pull request by Tigraine**
Changed the modules around to have a bitmovin-analytics-exoplayer and bitmovin-analyltics-bitmovin-player package

### CODE REVIEW

**Comment by zachmanc** [comment]
opening it up i get these build errors? anything specific to your environment that you changed?
image

**Comment by Tigraine** [comment]
Could this be a problem because of the 2.8.+ dependency?

I didn't change anything specifically but gradle has been a huge PITA :(.

File: collector/src/main/java/com/bitmovin/analytics/data/EventData.java [hunk 1]
```diff
@@ -75,7 +75,7 @@ public EventData(BitmovinAnalyticsConfig bitmovinAnalyticsConfig, String impress
         this.path = bitmovinAnalyticsConfig.getPath();
         this.experimentName = bitmovinAnalyticsConfig.getExperimentName();
         this.playerTech = Util.getPlayerTech();
-        this.setUserAgent(Util.getUserAgent(bitmovinAnalyticsConfig.getContext()));
+        this.setUserAgent("unknown"); //Util.getUserAgent(bitmovinAnalyticsConfig.getContext()));
```
Comment (hawk23) [inline collector/src/main/java/com/bitmovin/analytics/data/EventData.java]: Do you plan to find a solution here or will it stay `"unknown"` for now?

File: collector/src/main/java/com/bitmovin/analytics/adapters/PlayerAdapterFactory.java [hunk 2]
```diff
@@ -0,0 +1,5 @@
+package com.bitmovin.analytics.adapters;
+
+public class PlayerAdapterFactory {
```
Comment (hawk23) [inline collector/src/main/java/com/bitmovin/analytics/adapters/PlayerAdapterFactory.java]: Leftover?

**Comment by Tigraine** [comment]
@hawk23 the two test apps in the repo do exactly that ..

**Pull request merged**